cmd/agent/subcommands/streamlogs/command.go

```diff
@@ -74,6 +110,14 @@ func streamLogs(log log.Component, config config.Component, cliParams *cliParams
 	urlstr := fmt.Sprintf("https://%v:%v/agent/stream-logs", ipcAddress, config.GetInt("cmd_port"))
 	return streamRequest(urlstr, body, func(chunk []byte) {
 		fmt.Print(string(chunk))
+
+		if streamLogParams.FilePath != "" {
+			err := writeToFile(streamLogParams.FilePath, string(chunk))
```

Comment:
would it make sense to leave the file open for the duration of `stream-logs` instead of open/write/close for every line?